featuretools/io/sql.py

```diff
@@ -0,0 +1,100 @@
+import pandas as pd
+import sqlalchemy as sa
+
+import featuretools as ft
+
+
+def from_sql(id, connection=None, connection_string=None, tables=None, index={},
```

Comment:
for the api, let's combine `connection` and `connection_string` into one argument. Then we can infer if it is a connection object or string. For example, see how pandas does it here: https://pandas.pydata.org/pandas-docs/stable/generated/pandas.read_sql_table.html#pandas.read_sql_table.